src/Components/ArticleDetails.jsx

```diff
@@ -94,6 +89,10 @@ function ArticleDetails() {
         </button>
       </div>
       <h3>Comments</h3>
+      <PostCommentForm
+        article_id={article_id}
+        onCommentAdded={handleCommentAdded}
```

Comment:
Would be better to declare handleCommentAdded in PostCommentform rather than passing it down as props